recipes/mender-connect/debian-master/control

```diff
@@ -7,8 +7,10 @@ Build-Depends: debhelper (>= 9)
 
 Package: mender-connect
 Architecture: any
-Depends: libc6 (>= 2.12), libglib2.0-0 (>=2.50), mender-client (>=2.5.0)
+Depends: libc6 (>= 2.12), libglib2.0-0 (>=2.50), mender-client (>3.1.0)
```

Comment:
Conclusions from f2f dicussion:
* Modify the build code to bump minor number (instead of bugfix number) for the "master" packages
* Set dependency to  `mender-client (>3.1.99)`